packages/grant-explorer/src/features/common/Navbar.tsx

```diff
@@ -59,6 +57,12 @@ export default function Navbar(props: NavbarProps) {
             </Link>
           </div>
           <div className="flex items-center gap-6">
+            <RoundsSubNav
+              onClick={(round: NavRound) => {
+                const url = `#/round/${round.chainId}/${round.id}`;
+                window.open(url, "_blank");
```

Comment:
Since we're using an app router:

1. what is the reason to go around it?
2. if it turns out to be a good reason, please add a comment afterwards to explain why the counter-intuitive choice is necessary

Reply:
Changed it back now as per product suggestion of not opening the round in new tab.